api/initialize.py

```diff
@@ -1,3 +1,55 @@
-from app.models import db
+from app.models import db, User, Election, Question
+from app.config import TEST_SERVER, MEMBERSHIP_API, SEED_DATA, TOKEN
+import json
+import requests
+import datetime
 
-db.create_all()
\ No newline at end of file
+db.drop_all()
+db.create_all()
+
+if(SEED_DATA):
+
+    User.query.delete()
+    db.session.commit()
+    print(TOKEN)
+    r = requests.get(MEMBERSHIP_API + 'api/v1/leaderboard', headers = TOKEN)
+
+    print(r.status_code)
+    print(r.json())
+
+    leaderboard_user = r.json()
+
+    users = leaderboard_user['leaderboard']
+
+    for u in users:
+        uName = u['firstName'] + " " + u['lastName']
+        
+        board = False #True if u['accessType'] == 'STAFF' else
```

Comment:
Pretty sure you can remove that comment anyway; we're not going to define board roles using the accessType property just yet:
```suggestion
        board = False
```